src/test/librbd/fsx.cc

```diff
@@ -2393,12 +2393,16 @@ test(void)
 			log4(OP_SKIPPED, OP_WRITESAME, offset, size);
 			goto out;
 		}
+		break;
         case OP_COMPARE_AND_WRITE:
                 /* compare_and_write not implemented */
                 if (!ops->compare_and_write) {
                         log4(OP_SKIPPED, OP_COMPARE_AND_WRITE, offset, size);
                         goto out;
                 }
+		break;
+	default:
```

Comment:
@dillaman Ok, I see.

So, like you said, shall we remove the default case in L2457 too?